packages/esm-appointments-app/src/homepage-tile/appointments-tile.resources.tsx

```diff
@@ -0,0 +1,18 @@
+import { openmrsFetch } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import dayjs from 'dayjs';
+
+const useAppointmentsData = () => {
+  const omrsDateFormat = 'YYYY-MM-DDTHH:mm:ss.SSSZZ';
+  const appointmentDate = dayjs(new Date().setHours(0, 0, 0, 0)).format(omrsDateFormat);
+
+  const url = `ws/rest/v1/appointment/all?forDate=${appointmentDate}`;
+
+  const { data, error, isLoading } = useSWR<{ data: Array<any> }>(url, openmrsFetch);
+
+  const responseData = data?.data;
+
+  return { data: responseData, error, isLoading };
```

Comment:
Was this intentional? Is `data?.data` have the necessary information to be displayed or do u have to chain `results` like the rest

Reply:
On the initial render `data.data` will throw an error because the network request hasn't been fulfilled yet.